python3: can't open file '/Volumes/ExternalDriveAther/nihchestxray/process_images.py': [Errno 2] No such file or directory
